lib/src/iterables/partition.dart

```diff
@@ -37,10 +37,14 @@ class _PartitionIterator<T> implements Iterator<List<T>> {
 
   final Iterator<T> _iterator;
   final int _size;
-  List<T> _current;
+  List<T>/*?*/ _current;
 
   @override
-  List<T> get current => _current;
+  List<T> get current {
+    if (_current == null) throw StateError('Must call moveNext before current');
```

Comment:
Use `{ }` for better formatting?

Reply:
Done. This will be deleted post nnbd either way.